cmd/fleet/main.go

```diff
@@ -34,6 +34,8 @@ import (
 	"fleet/internal/pkg/signal"
 )
 
+const kPolicyThrottle = time.Millisecond * 5
```

Comment:
assuming this will be configurable?

Reply:
Whats the benefit of it being configurable?

Reply:
👍